src/pages/MainLayout/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import ButtonBox from "../../components/ButtonBox/index.jsx";
-import Questions from "../../components/Questions/index.jsx";
+import Questions from "../../components/PersonalInformation/index.jsx";
```

Comment:
If I'm not mistaken you can also do this:
```jsx
import ButtonBox from "../../components/ButtonBox";
import Questions from "../../components/PersonalInformation";
```

Reply:
doesn't work for me